packages/plugin-core/src/test/suite-integ/Extension.test.ts

```diff
@@ -68,15 +71,38 @@ function lapsedMessageTest({
 
 suite("Extension", function () {
   let homeDirStub: SinonStub;
+  let keybindings: any;
+  let backupKeybindingConfigPath: any;
+  const userConfigDir = VSCodeUtils.getCodeUserConfigDir()[0];
+  const keybindingConfigPath = [
+    userConfigDir,
+    "keybindings.json"
+  ].join("");
 
   const ctx: ExtensionContext = setupBeforeAfter(this, {
     beforeHook: async () => {
       await resetCodeWorkspace();
       await new ResetConfigCommand().execute({ scope: "all" });
       homeDirStub = TestEngineUtils.mockHomeDir();
+      if (!fs.existsSync(keybindingConfigPath)) {
```

Comment:
instead of overwriting system keybindings, lets stub this into `tmpDir`. see [[Stubbing Mutating Disk Changes|dendron.dev.qa#stubbing-mutating-disk-changes]]